src/chainlit/config.py

```diff
@@ -103,6 +103,31 @@ class CodeSettings:
     lc_factory: Optional[Callable[[], Any]] = None
     lc_rename: Optional[Callable[[str], str]] = None
 
+    def validate(self):
+        requires_one_of = [
+            "lc_factory",
+            "on_message",
+            "on_chat_start",
+        ]
+
+        mutually_exclusive = ["lc_factory"]
```

Comment:
Is there one item missing in this array?

Reply:
No but there will be more in with the next integrations, typically llama_index